components/email/src/Email.svelte

```diff
@@ -587,22 +587,27 @@
     const existingDraft = activeThread?.drafts?.find(
       (draft) => draft.reply_to_message_id === message.id,
     );
+    const subject = `Fwd: ${message.subject}`;
+    const value = {
+      reply_to_message_id: message.id,
+      subject: subject,
+      body: message.body,
+    };
     if (existingDraft) {
-      dispatchDraftEvent(event, existingDraft);
+      value.body = existingDraft.body;
+      dispatchEvent("forwardClicked", {
+        event,
+        message: { ...existingDraft, subject },
+        thread: activeThread,
+        value,
+      });
     } else {
       //Create new message
-      const subject = `Fwd: ${message.subject}`;
-      const value = {
-        reply_to_message_id: message.id,
-        subject: subject,
-        body: message.body,
-      };
       dispatchEvent("forwardClicked", {
         event,
-        message,
+        message: { ...message, subject },
         thread: activeThread,
         value,
-        focus_body_onload: false,
```

Comment:
was this removed on purpose?

Reply:
Yes, this is removed on purpose. `focus_body_onload` is a prop of composer component, so its value should not be populated from Email component.

Instead its added in the demo index.html when listening to events.